region/brinstar/blue.json

```diff
@@ -78,7 +78,18 @@
           "to": [
             {
               "id": 5,
-              "requires": [ "canPassBombPassages" ],
+              "requires": [
+                {"or":[
+                  "canPassBombPassages",
+                  {"and":[
+                    {"canShineCharge": {
+                      "usedTiles": 18,
+                      "shinesparkFrames": 0
```

Comment:
We should probably default shinesparkFrames to 0 in schema.

Reply:
Maybe but I don't know how to put logical elements in the schema so it's not in there.

Also, I'd just rather put 0 down manually because it's not a very common case, so writing it down underlines that yes, this is really what I intended.

Reply:
Do you want to add a commit to this with the 0's typed out? Like you said it's not that awfully many. Either that or just merge it.

Reply:
I'd do that as its own thing. No reason why doing that everywhere should hold up a PR of two strats in one room.